go.mod

```diff
@@ -7,30 +7,31 @@ go 1.23
 require (
 	github.com/onsi/ginkgo/v2 v2.19.0
 	github.com/onsi/gomega v1.33.1
-	k8s.io/apimachinery v0.31.0
-	k8s.io/client-go v0.31.0
+	k8s.io/apimachinery v0.31.1
+	k8s.io/client-go v0.31.1
```

Comment:
question, why do you bump dependencies in this PR?

can you do it in a separate PR so that we can merge firstly?

Reply:
Sure, it was an automated IDE bump. I will remove it from this PR. 
Thanks!